### PR TITLE
Fix refresh user token

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:** The way the user's login token was being reloaded is bad and does not work correctly. At each request, the token is reloaded, and if the user spends more than an hour without making a request, the token will expire and the user is de-logged.

**Solution:** Update the token in a certain time interval, before the token expires, instead of updating in every request.

**TODO/FIXME:** n/a

### CODE REVIEW

File: ecis [hunk 1]
```diff
@@ -183,7 +183,7 @@ case "$1" in
         set_support_url local $SUPPORT_LOCAL
         set_frontend_url local $FRONTEND_LOCAL
 
-        dev_appserver.py $APP_YAML $FRONTEND_YAML $BACKEND_YAML $SUPPORT_YAML $WORKER_YAML $2
+        dev_appserver.py $APP_YAML $FRONTEND_YAML $BACKEND_YAML $SUPPORT_YAML $WORKER_YAML -A development-cis $2
```
Comment (raonismaneoto) [inline ecis]: It shouldn't be here, right?

File: frontend/auth/authService.js [hunk 2]
```diff
@@ -8,21 +8,51 @@
...
+                });
+            } else if (tokenLoaded) {
+                return userInfo.accessToken;
+            }
+
+            return loadTokenPromise;
+        };
+
+        function getIdToken(user) {
+            user.getIdToken(true). then(function(userToken) {
```
Comment (raonismaneoto) [inline frontend/auth/authService.js]: there is a white space between then and the dot.

File: frontend/auth/authService.js [hunk 3]
```diff
@@ -8,21 +8,51 @@
...
+                if (userInfo) {
+                    userInfo.accessToken = userToken;
+                    service.save();
+                    console.log(userToken);
```
Comment (raonismaneoto) [inline frontend/auth/authService.js]: remove

File: frontend/auth/authService.js [hunk 4]
```diff
@@ -8,21 +8,51 @@
...
+                }
+
+                if (resolveTokenPromise)
+                    resolveTokenPromise(userToken);
+                    resolveTokenPromise = null;
+
+                tokenLoaded = true;
+            })
```
Comment (raonismaneoto) [inline frontend/auth/authService.js]: Is it missing a semicolon?

Comment (Luiz-FS) [reply frontend/auth/authService.js]: Fixed.

**Pull request merged**